doc/modules/ROOT/pages/tutorials/gds-sessions-self-managed.adoc

```diff
@@ -1,6 +1,6 @@
 // DO NOT EDIT - AsciiDoc file generated automatically
 
-= GDS Sessions (For Self Managed DB)
+= GDS Sessions - For Self Managed DB
```

Comment:
Was this updated from the notebook?

Reply:
now it is :)

Reply:
Nice! Nitpick: It would be better to skip either the "For" ("GDS Sessions - Self Managed DB") or the dash ("GDS Sessions for Self Managed DB"). Same applies to AuraDB below.
Also, I think that "self-managed" should have a hyphen.